lib/lroPollStrategy.ts

```diff
@@ -398,7 +440,7 @@ class LocationLROPollStrategy extends LROPollStrategy {
     } else {
       getResourceRequestUrl = initialRequest.url;
     }
-    return this.updateState(getResourceRequestUrl);
+    return this.updateState(getResourceRequestUrl, true);
```

Comment:
the typescript compiler codebase sometimes uses block comments to label arguments, e.g. `/* shouldDeserialize */ true`. It might be helpful here (but I think it's not refactor safe...)